inc/Engine/Preload/Database/Queries/RocketCache.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+namespace WP_Rocket\Engine\Preload\Database\Queries;
+
+use WP_Rocket\Dependencies\Database\Query;
+
+class RocketCache extends Query {
```

Comment:
```suggestion
class Cache extends Query {
```

I think it's ok to name the class like this directly